app/models/PayeAtsData.scala

```diff
@@ -24,7 +24,12 @@ case class PayeAtsData(
   summary_data: Option[DataHolder],
   income_data: Option[DataHolder],
   allowance_data: Option[DataHolder],
-  gov_spending: Option[GovernmentSpendingOutputWrapper])
+  gov_spending: Option[GovernmentSpendingOutputWrapper]) {
+  def isWelshTaxPayer: Boolean =
```

Comment:
Missing tests, I would also add a comment here mentioning that using `scottish_income_tax` is not a bug and we expect the field to change in the backend in 2021.

Reply:
done